src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx

```diff
@@ -36,6 +36,9 @@ export default function CSVFileImport({url, title}: CSVFileImportProps) {
         url,
         params: {
           name: encodeURIComponent(file.name)
+        },
+        headers: {
+          Authorization: `Basic ${localStorage.getItem('authorization_token')}`,
```

Comment:
not sure that we need provide Authorization header at all if we don't have anything in authorization_token

I mean is it valid at all to send `Authorization: "Basic null"`